lib/hooks/tests/use-error-codes.test.ts

```diff
@@ -0,0 +1,79 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useReferenceData } from '../../api/reference-data/v1';
+import locale from '../../locale';
+import { useErrorCodes } from '../use-error-codes';
+
+jest.mock('../../api/reference-data/v1');
```

Comment:
Do you need to mock, or could you use `msw`?
@hackney/mtfh-test-utils 👀

Reply:
Replaced all together with `msw`